apps/arkmarket/src/hooks/usePrices.ts

```diff
@@ -0,0 +1,46 @@
+import { useQuery } from "react-query";
+import { formatEther } from "viem";
+
+import getPrices from "~/lib/getPrices";
+
+interface ConvertInUsdParams {
+  token?: "ethereum" | "starknet";
+  amount?: bigint;
+}
+
+export default function usePrices() {
+  const { data, isLoading, isError, error } = useQuery(["prices"], getPrices, {
+    refetchInterval: 15_000,
+    staleTime: 15_000,
+  });
+
+  const convertInUsd = ({
+    token = "ethereum",
+    amount = BigInt(0),
+  }: ConvertInUsdParams) => {
+    if (!data) {
+      return 0;
+    }
+
+    const amountInEther = parseFloat(formatEther(amount));
+    const price = data[token].price;
+
+    return (amountInEther * price).toLocaleString("en-us", {
```

Comment:
We can have `undefined` instead of `"en-us"`, so we can use the local "language"